js/Landing.jsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styled from 'styled-components';
+
+const Wrapper = styled.div`
+   color: white;
+`;
+
+const LandingNavbar = styled.div`
+  h2{
+    float:left;
+    margin:2px;
+    font-weight: normal;
+  }
+  a{
+    float:right;
+    margin:5px;
+    font-weight: bolder;
+  }
+  font-family: Roboto
+
+`;
+
+const LandingHeader = styled.div`
+text-align: center;
+word-wrap: break-word;
+padding-top: 50px;
+padding-bottom: 10vh;
+font-family: 'Questrial';
+font-size: 150%;
+button{
+  font-family: 'Questrial';
+  font-weight: bold;
+  font-size: 100%;
+  background-color: #F7b733;
```

Comment:
we decided to put the button attributes in the global css file instead because it's easier and we therefore don't need to pass it to props each time we want to use it